rep-2000.rst

```diff
@@ -1,20 +1,69 @@
 REP: 2000
-Title: ROS 2 Target Platforms
-Author: Mikael Arguedas <mikael@openrobotics.org>, Steven! Ragnarok <steven@openrobotics.org>
+Title: ROS 2 Releases and Target Platforms
+Author: Mikael Arguedas <mikael@openrobotics.org>, Steven! Ragnarok <steven@openrobotics.org>, Dirk Thomas <dthomas@openrobotics.org>
 Status: Active
 Type: Informational
 Content-Type: text/x-rst
-Created: 10-Apr-2018, 24-Apr-2018, 21-May-2018, 10-Dec-2018, 20-May-2019, 21-May-2019, 29-Aug-2019
+Created: 10-Apr-2018, 24-Apr-2018, 21-May-2018, 10-Dec-2018, 20-May-2019, 21-May-2019, 29-Aug-2019, 25-Feb-2020
 
 
 Abstract
 ========
 
-This REP defines target platforms for each ROS 2 Distribution Release.
+This REP defines timeline for future ROS 2 releases as well as the targeted platforms for each specific one.
 We define platforms to include both operating system releases (e.g. Ubuntu Xenial (16.04 LTS)) as well as major language releases (e.g. Python 3.5).
 The target platforms represent the set on which all core stacks are expected to work.
 Exceptions can be made for stacks that are intentionally platform-specific.
 
+Release Schedule
+================
+
+Note: the following applies to ROS 2 releases *after* Foxy Fitzroy.
+Before, releases were made more frequently but with shorter support due to the fact that many foundational parts of ROS 2 were still heavily iterated on.
+
+TL;DR ROS 2 release schedule post Foxy:
+* May 2021: G Turtle: non-LTS release, supported for 1.5 years
+* May 2022: H Turtle: LTS release, supported for 5 years
+* May 2023: I Turtle: non-LTS release, supported for 1.5 years
+* May 2024: J Turtle: LTS release, supported for 5 years
+* and so on, alternating annually between LTS and non-LTS releases
+
+Frequency
+---------
+
+New ROS 2 releases are published in a time based fashion **every 12 months**.
+The rationale is that a shorter cycle (like 6 months) results in a significant overhead and many active releases at the same time (assuming they have the same support length).
+On the other hand a longer cycle (like 2 years) is too long for users to wait for new features to be available in a ROS 2 release.
+
+Targeted Platforms
+------------------
+
+Since regular Ubuntu releases are only supported for 9 months ROS 2 is not targting those.
+A single ROS 2 distribution only **targets one Ubuntu LTS**.
+The rationale is that supporting two LTS versions - which means 2-year-different versions of upstream dependencies - is a tremendous overhead and sometimes even impossible.
```

Comment:
The rationale given here makes sense. However, Ubuntu releases every 6 months. It's unclear how ROS 2 releases will be supported on Ubuntu's non-LTS releases.

Reply:
> It's unclear how ROS 2 releases will be supported on Ubuntu's non-LTS releases.

This is explicitly mentioned two lines above - in line 41.

> Not at all?

Correct.

Reply:
Ah, the "regular" wording threw me. Very good.

Reply:
Haha, I just tried to stick to the wording Canonical uses for it in their docs :wink:

Reply:
/me hides